scripts/ci/lib.sh

```diff
@@ -152,6 +152,24 @@ get_central_debug_dump() {
 
     roxctl -e "${API_ENDPOINT}" -p "${ROX_PASSWORD}" --insecure-skip-tls-verify central debug dump --output-dir "${output_dir}"
     ls -l "${output_dir}"
+
+    local metrics_output
+    local csv_output
+    metrics_output="$(mktemp --suffix=.prom)"
+    csv_output="$(mktemp --suffix=.csv)"
+    unzip -p "${output_dir}/*.zip" metrics-1 > "${metrics_output}"
+
+    go install github.com/stackrox/prometheus-metric-parser@latest
```

Comment:
Please pin this and set up updates via dependabot. Moving targets usually end up being a real PITA.

Reply:
I'll add a todo as it's also used in other place https://github.com/stackrox/stackrox/blob/2333f8046ebacb348fe5ca33ee327cb0f5c3f69b/tests/e2e/run-scale.sh#L77-L80

Reply:
Maybe we should move it into stackrox :thinking: